lib/network.py

```diff
@@ -1457,6 +1461,8 @@ def try_to_recover(err):
         assert not xout
         for interface in wout:
             interface.send_requests()
```

Comment:
Nit/lament: Return value never checked here. 😢

Reply:
Oh yeah, great point... sheesh lmao. Here I was taking care of this return value nobody uses hehe

Reply:
OK I just put up 3267189c6 to address this ... but to be honest I can't find any practical way to trigger any way for send_requests to return False.